### PR TITLE
Create seperate crud method to fetch positions independent on status

### PR DESCRIPTION
**Pull request by greatest0fallt1me**
#430

### CODE REVIEW

**Comment by greatest0fallt1me** [comment]
1. Created a new CRUD method called `get_all_positions_by_wallet_id` to fetch positions regardless of their status and updated the `get_user_positions` endpoint to use this new CRUD method.
2. Removed pagination from the dashboard endpoint as it is no longer needed.
3. Verified that all test cases still pass after the code changes.
4. Updated the existing test cases to use the new CRUD method to ensure user positions are fetched regardless of their status

**Comment by greatest0fallt1me** [comment]
@iamnovichek Can you please review this PR?

File: web_app/db/crud/position.py [hunk 1]
```diff
@@ -97,6 +97,42 @@ def get_positions_by_wallet_id(
...
+                    db.query(Position)
+                    .filter(
+                        Position.user_id == user.id,
+                    )
+                    .offset(start)
+                    .limit(limit)
+                    .all()
+                )
+                # Convert positions to a list of dictionaries
+                positions_dict = [
```
Comment (iamnovichek) [inline web_app/db/crud/position.py:127]: Rename it to `position_dicts`

**Review by iamnovichek — CHANGES_REQUESTED** [review]
GJ

**Pull request merged**